src/Blorc.Core/Bindings/Binding.cs

```diff
@@ -139,11 +141,11 @@ private void Initialize()
         protected override void Uninitialize()
         {
             _source.ValueChanged -= OnSourceValueChanged;
-            _source.Dispose();
+            _source?.Dispose();
```

Comment:
if we nullcheck this, we should nullcheck the line before as well. It's better to write:

```
var source = _source;
if (source is not null)
{
    source.ValueChanged -= OnSourceValueChanged;
    source.Dispose();
}
```